### PR TITLE
Revise TD doc URL

### PR DESCRIPTION
**Pull request by akito19**
We changed our documentation from `support.treasuredata.com` to `tddocs.atlassian.net`. So I revised the old URLs.

### CODE REVIEW

**Comment by akito19** [comment]
I didn't fix URL for https://github.com/treasure-data/treasure-boxes/blob/master/td/bigquery/README.md#supplemental because we haven't prepared explanations about the options.

File: integration-box/pelion-device-management/README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 ## Overview
 
-This Box demonstrates a simple way of pulling resource values from PDM to TD via [`ConnectAPI`](https://www.pelion.com/docs/device-management/current/service-api-references/device-management-connect.html) called by [PDM Python Client](https://github.com/ARMmbed/mbed-cloud-sdk-python) running on the [Treasure Workflow Custom Scripting](https://support.treasuredata.com/hc/en-us/articles/360026713713-Introduction-to-Custom-Scripts) capability.
+This Box demonstrates a simple way of pulling resource values from PDM to TD via [`ConnectAPI`](https://www.pelion.com/docs/device-management/current/service-api-references/device-management-connect.html) called by [PDM Python Client](https://github.com/ARMmbed/mbed-cloud-sdk-python) running on the [Treasure Workflow Custom Scripting](https://tddocs.atlassian.net/wiki/spaces/PD/pages/1084269/Custom%2BScripts) capability.
```
Comment (chezou) [inline integration-box/pelion-device-management/README.md:9]:
```suggestion
This Box demonstrates a simple way of pulling resource values from PDM to TD via [`ConnectAPI`](https://www.pelion.com/docs/device-management/current/service-api-references/device-management-connect.html) called by [PDM Python Client](https://github.com/ARMmbed/mbed-cloud-sdk-python) running on the [Treasure Workflow Custom Scripting](https://tddocs.atlassian.net/wiki/spaces/PD/pages/1084247/Introduction+to+Custom+Scripts) capability.
```

File: integration-box/pyspark/README.md [hunk 2]
```diff
@@ -15,4 +15,4 @@ $ td wf start td-spark pyspark --session now
 
 ## Further Readings
 
-- [td-spark Official doc](https://support.treasuredata.com/hc/en-us/articles/360001487167-Apache-Spark-Driver-td-spark-FAQs)
+- [td-spark Official doc](https://tddocs.atlassian.net/wiki/spaces/PD/pages/1082513/Apache+Spark+Driver+td-spark+FAQs)
```
Comment (chezou) [inline integration-box/pyspark/README.md:18]: Can be better to point https://treasure-data.github.io/td-spark/ like:

```suggestion
- [td-spark Official doc](https://treasure-data.github.io/td-spark/)
```

File: machine-learning-box/recommendation/README.md [hunk 3]
```diff
@@ -31,14 +31,14 @@ $ td wf push recommendation # push workflow to TD
...
 
 [<img src="docs/img/capture.png" alt="capture" max_height=300 />](http://showterm.io/31b8df49efcfbc2bfc5ef#fast)
 
 ### Workflow with PySpark
 
-You also can try [Spark.ml collaborative filtering](https://spark.apache.org/docs/2.4.0/ml-collaborative-filtering.html) recommendation for top-k item recommendation. Ensure both [Python Custom Scripting](https://support.treasuredata.com/hc/en-us/articles/360026713713-Introduction-to-Custom-Scripts) and [td-spark](https://support.treasuredata.com/hc/en-us/articles/360000716627-Apache-Spark-Driver-td-spark-Release-Notes) enabled.
+You also can try [Spark.ml collaborative filtering](https://spark.apache.org/docs/2.4.0/ml-collaborative-filtering.html) recommendation for top-k item recommendation. Ensure both [Python Custom Scripting](https://tddocs.atlassian.net/wiki/spaces/PD/pages/1084269/Custom%2BScripts) and [td-spark](https://tddocs.atlassian.net/wiki/spaces/PD/pages/1082542/Using+Apache+Spark+Driver+TD-Spark+in+your+Spark+Environment) enabled.
```
Comment (chezou) [inline machine-learning-box/recommendation/README.md:41]:
```suggestion
You also can try [Spark.ml collaborative filtering](https://spark.apache.org/docs/2.4.0/ml-collaborative-filtering.html) recommendation for top-k item recommendation. Ensure both [Python Custom Scripting](https://tddocs.atlassian.net/wiki/spaces/PD/pages/1084247/Introduction+to+Custom+Scripts) and [td-spark](https://tddocs.atlassian.net/wiki/spaces/PD/pages/1082542/Using+Apache+Spark+Driver+TD-Spark+in+your+Spark+Environment) enabled.
```

**Comment by akito19** [comment]
@chezou Thank you for reviewing. I fixed those URLs.

**Comment by akito19** [comment]
@mickeey2525 Do you think about this change?

**Pull request merged**